src/coreclr/tools/Common/TypeSystem/Common/FieldLayoutAlgorithm.cs

```diff
@@ -88,7 +88,7 @@ public struct ComputedInstanceFieldLayout
         /// If Offsets is non-null, then all field based layout is complete.
         /// Otherwise, only the non-field based data is considered to be complete
         /// </summary>
-        public FieldAndOffset[] Offsets;
+        public FieldAndOffset[]? Offsets;
```

Comment:
I would just turn off nullable for the type system project. These files are included from a bunch of places that are not on the nullable plan, so it will be a bunch of unnecessary work. Type system is better off without nullable anyway.